### PR TITLE
FSET-1906: Add endpoint to mark a scheme's sift result as red

### PR DESCRIPTION
**Pull request by henricook**

### CODE REVIEW

File: app/repositories/sift/ApplicationSiftRepository.scala [hunk 1]
```diff
@@ -301,5 +303,53 @@ class ApplicationSiftMongoRepository(
...
     }
   }
+
+  def fixDataByRemovingSiftEvaluation(applicationId: String): Future[Unit] = {
+    val query = BSONDocument("applicationId" -> applicationId)
+
+    val updateOp = bsonCollection.updateModifier(
+      BSONDocument(
+        "$set" -> BSONDocument("applicationStatus" -> ApplicationStatus.SIFT),
+        "$unset" -> BSONDocument(s"testGroups.$phaseName" -> "")
```
Comment (fayimora) [inline app/repositories/sift/ApplicationSiftRepository.scala]: @henricook this un-sets the entire sift phase. I think we want to remove the result for a particular scheme here.

Comment (henricook) [reply app/repositories/sift/ApplicationSiftRepository.scala]: The ticket hasn't specified a particular scheme, and i believe we talked about removing their whole evaluation so they could just be re-sifted

Comment (henricook) [reply app/repositories/sift/ApplicationSiftRepository.scala]: The ticket contains two candidates, one who needs an individual scheme red'ing and one who needs to be rolled back from AC_AWAITING_ALLOCATION

Comment (henricook) [reply app/repositories/sift/ApplicationSiftRepository.scala]: The currentSchemeStatus also needs resetting, i've had some crossover here so took care of it in the PR for FSET-1909

**Pull request merged**